src/main/java/picard/fingerprint/CrosscheckFingerprints.java

```diff
@@ -247,9 +260,20 @@
             "Need only include the samples that change. " +
             "Values in column 1 should be unique. " +
             "Values in column 2 should be unique even in union with the remaining unmapped samples. " +
-            "Should only be used with SECOND_INPUT. ", optional = true)
+            "Should only be used with SECOND_INPUT. ", optional = true, mutex = {"INPUT_SAMPLE_FILE_MAP"})
     public File INPUT_SAMPLE_MAP;
 
+    @Hidden
+    @Argument(doc = "A tsv with two columns representing " +
+            "the sample as it should be used for comparisons to SECOND_INPUT (in the first column) and  " +
+            "the source file (in INPUT) for the fingerprint (in the second column). " +
```

Comment:
This is a slightly confusing use case. Is the situation here that you have many INPUTs but only one (multi-sample) SECOND_INPUT?